src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -670,21 +670,26 @@ void AsynchronousMetrics::update(std::chrono::system_clock::time_point update_ti
         {
             Int64 amount = total_memory_tracker.get();
             Int64 peak = total_memory_tracker.getPeak();
-            Int64 new_amount = data.resident;
+            Int64 rss = data.resident;
 
-            Int64 difference = new_amount - amount;
+            Int64 rss_drift = rss - amount;
+            Int64 difference = rss_drift - last_logged_rss_drift;
```

Comment:
What does it mean?

Reply:
I just try to keep the logic of logging as it was before.
Previously, we logged the change between RSS and MemoryTracking every time it became bigger then 1MB. Also, previously we set MemoryTracking to RSS. I removed this logic and keep RSS and MemoryTracking separately.
In new terms, we have a drift between RSS and MemoryTracking and log it when this drift changed in more then in 1MB.